alignak/http/generic_interface.py

```diff
@@ -282,17 +284,18 @@ def get_raw_stats(self):
         app = self.app
         res = {}
 
-        for sched_id in app.schedulers:
-            sched = app.schedulers[sched_id]
-            lst = []
-            res[sched_id] = lst
-            for mod in app.q_by_mod:
-                # In workers we've got actions send to queue - queue size
-                for (q_id, queue) in app.q_by_mod[mod].items():
-                    lst.append({
-                        'scheduler_name': sched['name'],
-                        'module': mod,
-                        'queue_number': q_id,
-                        'queue_size': queue.qsize(),
-                        'return_queue_len': app.get_returns_queue_len()})
+        if hasattr(app, 'schedulers'):
```

Comment:
For one or two (not keep in my memory) daemons, this attribute not exist, so prevent 500 error in cherrypy